### DISCUSSION TITLE
New board from Atlas Scientific

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#885] [Nixie] [answered]
It looks like the T3 board has been replaced with a new board for about half the price.  Of course I bought the T3 a couple of weeks ago :-(

https://atlas-scientific.com/electrical-isolation/i3-interlink/

**Comment by rstrouse** [comment, marked as answer]
Well look a that and they got rid of that goofball 40 pin through connector.

**Reply by FlaMike** [reply]
Thanks--for the benefit of others!

I hate that 40 pin connector.  I hoped to use it to power a 5 volt fan, but decided to use the Pi's USB port & hack some wires to be able to have thermostatic control via a relay because the connector can't use "normal" Dupont connectors.  What the heck were they thinking?

**Reply by rstrouse** [reply]
Yeah I have done a couple now with the Sequent Smart fan.  Soooo much easier.

**Reply by FlaMike** [reply]
I guess Atlas liked being at the "top" of the stack since you couldn't put their card anywhere else.  Sigh.

**Reply by FlaMike** [reply]
@rstrouse   After seeing your post above, I have a better understanding of your response in my fan query thread the other day.  I'm not using a Sequent Smart Fan, but since my Atlas board is a T3 your approach makes even more sense.  PWM & a CPU cooler would be nice to have, but I'll stick with the 5 volt 120mm case fan that's installed running at "full speed" unless it starts broiling chicken :-)

**Reply by truvec** [reply]
Hmm. I just used some female extension headers to give proper clearance where necessary. Does this violate something?!
![IMG_2102](https://github.com/tagyoureit/nodejs-poolController/assets/239790/adbb39b1-7886-461e-b8c9-4c217688d87e)
![IMG_2101](https://github.com/tagyoureit/nodejs-poolController/assets/239790/f1d6589d-fdd1-4359-b9b1-1319239b0973)

**Reply by tagyoureit** [reply]
The board is also i2c only, which is fine for us.  

And there is no issue using the extension headers.  I do the same.

**Reply by FlaMike** [reply]
If I had any extension headers handy, I would have used them as well.  In my case, I wish I had known about the new board.  I would have waited a couple of weeks to order it, saved about $50.00 and had no issue with the headers that came with the board :-(

Such is  life!  All good here.

**Comment by ch0use** [comment]
I have ordered the i3 InterLink board and a Pressure sensor, but I can't get anything new to show up in i2cdetect -y 1. There is nothing detected. This is on a raspberry pi 5 with a PoE HAT, and then the i3 stacked on top of it. There is a green blinking LED on the pressure sensor. Not sure if that means it's in UART mode. Anybody have any ideas?

**Reply by rstrouse** [reply]
Green means UART mode you need it to be blue for I2c mode.  The process to switch it is in the datasheet.

https://files.atlas-scientific.com/EZO-PRS-Datasheet.pdf

**Comment by ch0use** [comment]
Thanks, yeah, I just came across that. Having a heck of a time establishing
a serial connection to it, trying with minicom against /dev/serial0 (linked
to /dev/ttyAMA10) and /dev/serial1 (linked to ttyS0). Can't get anything to
come back, using 9600 8N1, no hardware/software flow control, carriage
return, everything.

**Reply by tagyoureit** [reply]
See page 34.  You can short TX and INT and it will switch it I2C mode.

**Comment by ch0use** [comment]
Took a few tries but that finally did it, thanks!

**Comment by ch0use** [comment]
One last thing, is there a way to tweak the reported pressure? It's off by
about 2 psi (gauge reads 13, sensor says 15). I already calibrated it with
the pool pump off.

**Reply by rstrouse** [reply]
Turn off the pump then click on the calibrate button when there should be 0psi.

**Comment by FlaMike** [comment]
Yay! Congratulations--persitance pays 👍
